package.json

```diff
@@ -78,20 +78,20 @@
     "lint:attw": "FORCE_COLOR=1 turbo lint:attw",
     "lint:fix": "FORCE_COLOR=1 turbo lint -- --fix",
     "bundlewatch": "turbo bundlewatch",
-    "format": "npx prettier --write .",
-    "format:check": "npx prettier --cache --check .",
+    "format": "prettier --write .",
+    "format:check": "prettier --cache --check .",
     "nuke": "./scripts/nuke.sh",
     "yalc:all": "for d in packages/*/; do echo $d; cd $d; yalc push --replace --sig; cd '../../'; done",
     "prepare": "husky install",
-    "changeset": "npx changeset",
+    "changeset": "changeset",
```

Comment:
There's no need to use `npx`, in fact I believe the current setup didn't allow for environment variables (https://brianchildress.co/environment-variables-using-npx/)